contracts/yield-source/ATokenYieldSource.sol

```diff
@@ -93,14 +96,17 @@ contract ATokenYieldSource is ERC20Upgradeable, IProtocolYieldSource, AssetManag
     initializer
     returns (bool)
   {
+    require(address(_aToken).isContract(), "ATokenYieldSource/aToken-not-contract-address");
     aToken = _aToken;
+
+    require(address(_lendingPoolAddressesProviderRegistry).isContract(), "ATokenYieldSource/lendingPoolRegistry-not-contract-address");
     lendingPoolAddressesProviderRegistry = _lendingPoolAddressesProviderRegistry;
 
     __Ownable_init();
     transferOwnership(_owner);
 
     __ERC20_init(_name,_symbol);
-    require(_decimals > 0, "ATokenYieldSource/decimals-gt-zero");
+    require(_decimals > 0, "ATokenYieldSource/decimals-not-greater-than-zero");
```

Comment:
keep it short- use just `gt`.  Every byte counts!

Reply:
Done: https://github.com/pooltogether/aave-yield-source/pull/19/commits/eedf50e1fcccdc8f5cd1cac66bef5bbd8e80d99c